source/playbooks/notifications-and-updates.rst

```diff
@@ -25,7 +29,13 @@ Status updates ensure that stakeholders remain informed about the playbook run
 4. Select **Post update** to post your status update.
 
 If the playbook has a defined broadcast channel, status updates are copied to the broadcast channel as a message from the Playbooks bot.
-The most recent status post will also appear in the right-hand sidebar of the run channel. To correct or remove a status post, edit or delete the post as needed. Note that status updates that are broadcast to another channel won’t be updated or removed if the original post is edited or deleted.
+
+The most recent status post will also appear in the right-hand sidebar of the run channel and in the timeline. To correct or remove a status post, edit or delete the post as needed. Note that status updates that are broadcast to another channel won’t be updated or removed if the original post is edited or deleted.
```

Comment:
Status post appear in the channel main body (not RHS) or in the Run details page body (RHS). The status post is maybe linked from the timeline but it's not shown there.

![CleanShot 2023-01-09 at 17 08 00@2x](https://user-images.githubusercontent.com/4096774/211356049-7d536e4c-0464-4e05-b765-cab1f87b66ea.png)